src/components/Toolbar.vue

```diff
@@ -218,10 +218,13 @@ export default {
       window.eventBus.$emit('zoomToFit');
     },
     exportData() {
-      this.thingWereSaving = 'Floorplan';
       this.showSaveModal = true;
       return this.$store.getters['exportData'];
     },
+    open3DPreviewer() {
+      localStorage.setItem("floorplan3DExport", JSON.stringify(application.$store.getters['exportData']));
+      window.open('3DViewer');
```

Comment:
this seems to be plenty fast, but if you wanted to avoid localstorage the postMessage api might be another option.

Reply:
In this case, I do want it to be stored in case the user refreshes the tab or reopens it.